### DISCUSSION TITLE
阶段一二中我是否需要实现用户交互

### DISCUSSION THREAD

**Discussion by VansamaZWT** [liusy58/NJU_AdvancedProgramming_SP22#175] [Ideas]
在阶段一二中，我很可能不引入GUI，那么在这种情况下，不同于python的逐行运行，C++的用户交互会变得十分困难。我是否可以只硬编码某些放置操作，然后通过日志播报这次操作固定的战斗过程？

**Comment by liusy58** [comment]
没有太明白你的问题

**Reply by VansamaZWT** [reply]
比如我想在游戏过程中放置一个塔，在没有gui信号和槽的情况下，我只能在函数里硬编码一段来调用放置函数，这样的话程序开始运行之后我不能在进行任何操作，这样可以吗

**Reply by liusy58** [reply]
可以，不过缺点是后续你的工作量会比较大。
